rosbag2_storage_default_plugins/include/rosbag2_storage_default_plugins/sqlite/sqlite_storage.hpp

```diff
@@ -42,7 +42,8 @@ class ROSBAG2_STORAGE_DEFAULT_PLUGINS_PUBLIC SqliteStorage
   : public rosbag2_storage::storage_interfaces::ReadWriteInterface
 {
 public:
-  SqliteStorage() = default;
+  SqliteStorage()
+       :no_of_inserts(0) {};
   ~SqliteStorage() override = default;
```

Comment:
Given that you specify a custom implementation of the destructor, the `default` keyword has to be removed here.